### ISSUE TITLE
xss

### ISSUE THREAD

**Issue by xcoderzach** [Amicus/end-dash#151]
Variable reaction is xss-able.  Should probably replace the call to 
`this.el.html` with `this.el.text`, unless something like, `data-htmlenable` exists.
I don't remember why we didn't fix this before, I think it had something to do with the call page, and maybe being lazy.

Try it for yourself 
1) go to http://www.enddash.com/ 
2) `<script>alert('xss')</script>` in the box
3) lulz

**Comment by tobowers** [comment]
Shouldn't this be the servers responsibility?

**Comment by xcoderzach** [comment]
Yes, but you don't want insecure defaults because then people blame the
library, not the insecure code.

**Comment by devmanhinton** [comment]
For the example on end-dash.com can we just add an 'change' event listener so that a script tag will not be run. Then in the misc section we can document that escaping is the server's responsibility.

?

**Comment by sethbannon** [comment]
@devmanhinton  it's enddash.com :wink:

**Comment by devmanhinton** [comment]
@sethbannon thanks :wink:

**Comment by xcoderzach** [comment]
I guess this will be easier to fix with the interpolation syntax, but it
still seems like you would want to be able to bind html to a model which is
bound to user input   (userInput -> model -> html)  without having to do
something wacky in between. (userInput -> cleanup -> model -> html)
